### ISSUE TITLE
"cannot clean working clone" error

### ISSUE THREAD

**Issue by emas80** [fluxcd/flux#2927]
Today, we were trying to release a new version of an application by using `fluxctl` and we are getting an error:
```
Deautomating workload XXX:deployment/YYY
Error: walking "/tmp/flux-working099870002" for chartdirs: walking "/tmp/flux-working099870002/.git/objects/d9/95d5cf4ba827267196204b95d5d4a854ae737e" for charts: lstat /tmp/flux-working099870002/.git/objects/d9/95d5cf4ba827267196204b95d5d4a854ae737e: no such file or directory
Run 'fluxctl deautomate --help' for usage.
```
We have been using Flux for quite sometime, we have many different clusters with a lot of Flux installation, and this is the first time I see anything like that.
I cannot even suggest any way to replicate it, as it's just happened out of the blue, and basically it was impossible to release anything to that specific Flux instance using fluxctl.

Killing the pod seems to have "fixed".

**Logs**
```
ts=2020-03-19T15:13:44.846900143Z caller=daemon.go:663 component=daemon error="cannot clean working clone: unlinkat /tmp/flux-working757088088/.git: directory not empty"
ts=2020-03-19T15:13:44.847037931Z caller=loop.go:151 component=sync-loop jobID=7aeb5976-ae6d-bcc5-df1a-3c67b9853b1f state=done success=false err="walking \"/tmp/flux-working757088088\" for chartdirs: walking \"/tmp/flux-working757088088/.git/objects/98/a3e49e736cd730799fe7293c6b21bdce5a871b\" for charts: lstat /tmp/flux-working757088088/.git/objects/98/a3e49e736cd730799fe7293c6b21bdce5a871b: no such file or directory"
````

**Additional context**

- Flux version: 1.18.0
- Git provider: github

**Comment by hiddeco** [comment]
Possible duplicate of #1558.

**Comment by kingdonb** [comment]
Seems to have been closed in #1559.
